pkg/controller/clusterpool/metrics.go

```diff
@@ -7,6 +7,40 @@ import (
 )
 
 var (
+	metricClusterDeploymentsAssignable = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "hive_clusterpool_clusterdeployments_assignable",
+		Help: "The number of ClusterDeployments ready to be claimed. Contributes to Size and MaxSize.",
+	}, []string{"clusterpool_namespace", "clusterpool_name"})
+	metricClusterDeploymentsClaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "hive_clusterpool_clusterdeployments_claimed",
+		Help: "The number of claimed ClusterDeployments, including those being deleted. Contributes to MaxSize.",
+	}, []string{"clusterpool_namespace", "clusterpool_name"})
+	metricClusterDeploymentsDeleting = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "hive_clusterpool_clusterdeployments_deleting",
+		Help: "The number of ClusterDeployments marked for or actively deprovisioning and deleting. Contributes to MaxConcurrent.",
+	}, []string{"clusterpool_namespace", "clusterpool_name"})
+	metricClusterDeploymentsInstalling = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "hive_clusterpool_clusterdeployments_installing",
+		Help: "The number of ClusterDeployments being added to the pool, in the process of installing. Contributes to Size, MaxSize, and MaxConcurrent.",
+	}, []string{"clusterpool_namespace", "clusterpool_name"})
+	// This is always assignable+installing. But it's the number we strive to keep equal to the pool size.
+	// Is it useful to include?
```

Comment:
I think that's a simple prom query so I vote for scraping it

Reply:
But OTOH, the consumer of these metrics would have to understand that internal business logic. Including knowing that it should include "standby" once that's implemented. I'm leaning towards keeping it :)